addons/jam/magwells_303B.hpp

```diff
@@ -1,4 +1,4 @@
-    class CBA_303B_LeeEn {};        // Lee Enfield
     class CBA_303B_BREN {};         // BREN gun
+    class CBA_303B_LeeEn {};        // Lee Enfield
```

Comment:
Just as a side note. We don't really need to shorten names to oblivion. Longer names don't hurt that much